aijson/flow.py

```diff
@@ -150,6 +158,17 @@ async def run(self, target_output: None | str = None) -> Any:
             raise RuntimeError("Failed to render result")
         return result
 
+    async def stream_all(self):
+        action_keys = list(self.action_config.flow.keys())
+        actions = {}
+        for action in action_keys:
+            action_outputs = []
+            stream_output = self.stream(action)
+            async for output in stream_output:
+                action_outputs.append(output)
```

Comment:
Instead, use `merge_iterators` to iterate all of them at once
On every change, yield the partial results